Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.7-slim
+WORKDIR /app
+
+COPY ./main.py /app/main.py
+COPY ./setup.py /app/setup.py
+COPY ./README.md /app/README.md
+RUN pip install -e ".[dev]"
+
+CMD ["python", "main.py"]
```

Comment:
Shouldn't this be `ENTRYPOINT` to be able to pass arguments like verbose or dry run?